src/core/services/layout/layout.js

```diff
@@ -239,4 +290,35 @@
 
     }
 
+    /**
```

Comment:
API addition:

*  Allows use of Attribute selectors for browsers that do not suffer performance degradations
*  Allows use of ngMaterial Layout/Flexbox CSS **without** loading the `angular-material.js` source.